### PR TITLE
Implement UI CV Package Filter tests

### PR DESCRIPTION
**Pull request by abalakh**
Part of #778 
Depends on #4052

### CODE REVIEW

**Comment by abalakh** [comment]
Test results (with #4052 applied):
```python
py.test tests/foreman/ui/test_contentview.py -k 'test_positive_add_package_inclusion_filter_and_publish or test_positive_add_package_exclusion_filter_and_publish or test_positive_remove_package_from_exclusion_filter or test_positive_update_inclusive_filter_package_version or test_positive_update_exclusive_filter_package_version or test_negative_add_same_package_filter_twice' -n 2 -v
[gw0] linux2 Python 2.7.10 cwd: /home/andrii/workspace/robottelo
[gw1] linux2 Python 2.7.10 cwd: /home/andrii/workspace/robottelo
[gw0] Python 2.7.10 (default, Sep  8 2015, 17:20:17)  -- [GCC 5.1.1 20150618 (Red Hat 5.1.1-4)]
[gw1] Python 2.7.10 (default, Sep  8 2015, 17:20:17)  -- [GCC 5.1.1 20150618 (Red Hat 5.1.1-4)]
gw0 [6] / gw1 [6]

scheduling tests via LoadScheduling

tests/foreman/ui/test_contentview.py::ContentViewTestCase::test_negative_add_same_package_filter_twice <- robottelo/decorators/__init__.py 
tests/foreman/ui/test_contentview.py::ContentViewTestCase::test_positive_add_package_inclusion_filter_and_publish <- robottelo/decorators/__init__.py 
[gw1] PASSED tests/foreman/ui/test_contentview.py::ContentViewTestCase::test_positive_add_package_inclusion_filter_and_publish <- robottelo/decorators/__init__.py 
tests/foreman/ui/test_contentview.py::ContentViewTestCase::test_positive_remove_package_from_exclusion_filter <- robottelo/decorators/__init__.py 
[gw0] PASSED tests/foreman/ui/test_contentview.py::ContentViewTestCase::test_negative_add_same_package_filter_twice <- robottelo/decorators/__init__.py 
tests/foreman/ui/test_contentview.py::ContentViewTestCase::test_positive_add_package_exclusion_filter_and_publish <- robottelo/decorators/__init__.py 
[gw1] PASSED tests/foreman/ui/test_contentview.py::ContentViewTestCase::test_positive_remove_package_from_exclusion_filter <- robottelo/decorators/__init__.py 
[gw0] PASSED tests/foreman/ui/test_contentview.py::ContentViewTestCase::test_positive_add_package_exclusion_filter_and_publish <- robottelo/decorators/__init__.py 
tests/foreman/ui/test_contentview.py::ContentViewTestCase::test_positive_update_inclusive_filter_package_version <- robottelo/decorators/__init__.py 
tests/foreman/ui/test_contentview.py::ContentViewTestCase::test_positive_update_exclusive_filter_package_version <- robottelo/decorators/__init__.py 
[gw0] PASSED tests/foreman/ui/test_contentview.py::ContentViewTestCase::test_positive_update_inclusive_filter_package_version <- robottelo/decorators/__init__.py 
[gw1] PASSED tests/foreman/ui/test_contentview.py::ContentViewTestCase::test_positive_update_exclusive_filter_package_version <- robottelo/decorators/__init__.py 

========================== 6 passed in 614.57 seconds ==========================
```

**Comment by abalakh** [comment]
Added the last test, test results:
```python
py.test tests/foreman/ui/test_contentview.py -k 'test_positive_update_filter_affected_repos'
================================================ test session starts ================================================
platform linux2 -- Python 2.7.10, pytest-3.0.3, py-1.4.31, pluggy-0.4.0
rootdir: /home/andrii/workspace/robottelo, inifile: 
plugins: xdist-1.13.1, repeat-0.2
collected 70 items 

tests/foreman/ui/test_contentview.py .

================================================ 69 tests deselected ================================================
===================================== 1 passed, 69 deselected in 203.24 seconds =====================================
```
PR is ready

File: robottelo/ui/contentviews.py [hunk 1]
```diff
@@ -5,6 +5,7 @@
 
 from robottelo.constants import FILTER_ERRATA_TYPE, FILTER_ERRATA_DATE
 from robottelo.ui.base import Base, UIError, UINoSuchElementError
+from robottelo.decorators import bz_bug_is_open
```
Comment (svtkachenko) [inline robottelo/ui/contentviews.py]: In alphabetical order `robottelo.decorators` should be before `robottelo.ui` :p

**Comment by abalakh** [comment]
Addressed nitpick and made a rebase

**Comment by oshtaier** [comment]
Had a discussion with @abalakh regarding this PR. He is going to add few small changes, but basically ACK

**Comment by abalakh** [comment]
@oshtaier comments addressed.
```python
============================= test session starts ==============================
platform linux2 -- Python 2.7.10, pytest-3.0.3, py-1.4.31, pluggy-0.4.0 -- /home/andrii/env/bin/python
cachedir: .cache
rootdir: /home/andrii/workspace/robottelo, inifile: 
plugins: xdist-1.13.1, repeat-0.2
gw0 I / gw1 I
[gw0] linux2 Python 2.7.10 cwd: /home/andrii/workspace/robottelo
[gw1] linux2 Python 2.7.10 cwd: /home/andrii/workspace/robottelo
[gw0] Python 2.7.10 (default, Sep  8 2015, 17:20:17)  -- [GCC 5.1.1 20150618 (Red Hat 5.1.1-4)]
[gw1] Python 2.7.10 (default, Sep  8 2015, 17:20:17)  -- [GCC 5.1.1 20150618 (Red Hat 5.1.1-4)]
gw0 [7] / gw1 [7]

scheduling tests via LoadScheduling

tests/foreman/ui/test_contentview.py::ContentViewTestCase::test_negative_add_same_package_filter_twice <- robottelo/decorators/__init__.py 
tests/foreman/ui/test_contentview.py::ContentViewTestCase::test_positive_add_package_inclusion_filter_and_publish <- robottelo/decorators/__init__.py 
[gw0] PASSED tests/foreman/ui/test_contentview.py::ContentViewTestCase::test_negative_add_same_package_filter_twice <- robottelo/decorators/__init__.py 
tests/foreman/ui/test_contentview.py::ContentViewTestCase::test_positive_add_package_exclusion_filter_and_publish <- robottelo/decorators/__init__.py 
[gw1] PASSED tests/foreman/ui/test_contentview.py::ContentViewTestCase::test_positive_add_package_inclusion_filter_and_publish <- robottelo/decorators/__init__.py 
tests/foreman/ui/test_contentview.py::ContentViewTestCase::test_positive_remove_package_from_exclusion_filter <- robottelo/decorators/__init__.py 
[gw0] PASSED tests/foreman/ui/test_contentview.py::ContentViewTestCase::test_positive_add_package_exclusion_filter_and_publish <- robottelo/decorators/__init__.py 
tests/foreman/ui/test_contentview.py::ContentViewTestCase::test_positive_update_exclusive_filter_package_version <- robottelo/decorators/__init__.py 
[gw1] PASSED tests/foreman/ui/test_contentview.py::ContentViewTestCase::test_positive_remove_package_from_exclusion_filter <- robottelo/decorators/__init__.py 
[gw0] PASSED tests/foreman/ui/test_contentview.py::ContentViewTestCase::test_positive_update_exclusive_filter_package_version <- robottelo/decorators/__init__.py 
tests/foreman/ui/test_contentview.py::ContentViewTestCase::test_positive_update_filter_affected_repos <- robottelo/decorators/__init__.py 
tests/foreman/ui/test_contentview.py::ContentViewTestCase::test_positive_update_inclusive_filter_package_version <- robottelo/decorators/__init__.py 
[gw1] PASSED tests/foreman/ui/test_contentview.py::ContentViewTestCase::test_positive_update_filter_affected_repos <- robottelo/decorators/__init__.py 
[gw0] PASSED tests/foreman/ui/test_contentview.py::ContentViewTestCase::test_positive_update_inclusive_filter_package_version <- robottelo/decorators/__init__.py 

========================== 7 passed in 873.07 seconds ==========================
```

**Comment by oshtaier** [comment]
ACK

**Pull request merged**